client/features/auth/Auth.jsx

```diff
@@ -45,13 +54,24 @@ class Auth extends React.Component {
     axios.post('/auth/user', this.state)
```

Comment:
Honestly, it's hard to even review code right now, so please  don't take this as anything other than an attempt at a helpful suggestion: have you considered allowing `/users` to be its own route, so that other features can add watch history, ratings, reviews etc. to user records?